UI/window-basic-interaction.cpp

```diff
@@ -215,6 +215,42 @@ static int TranslateQtMouseEventModifiers(
 	return modifiers;
 }
 
+static int TranslateQtNavigationKeys(
+	QKeyEvent *event)
```

Comment:
Style:  There's no real reason to line break for this argument list since the previous line doesn't exceed the line column limit.  Make this function definition one line.